drizzlepac/align.py

```diff
@@ -39,29 +39,33 @@
 # ----------------------------------------------------------------------------------------------------------
 
 
-def check_and_get_data(input_list, **pars):
+def check_and_get_data(input_list: list, **pars: object) -> list:
```

Comment:
Just curious why this function signature is setup this way now.  It is helpful, but it is also different than most functions.  As a newbie, I had to look this up!

Reply:
I was using PyCharm's suggestions for updating the signature of this function in an attempt to get Sphinx to successfully parse this function API.